testbed/www/js/index.js

```diff
@@ -55,7 +55,9 @@ app.initialize();
 
 function DeepLinkHandler(data)
 {
-    alert('Initialize: ' + JSON.stringify(data));
+    if (data) {
```

Comment:
With the change of returning a string, this will always return something. I don't think we want that behavior. I'd expect separate logic in here that says
```javascript
if (data) {
  alert("Initialize: " + JSON.stringify(data));
} else {
  alert("No data was received");
}
```